### DISCUSSION TITLE
Is my RE music usage fair enough for you?

### DISCUSSION THREAD

**Discussion by vipertechofficial** [redeclipse/base#1349] [Help and Support] [answered]
![Screenshot RE](https://user-images.githubusercontent.com/74778466/169098600-ad55149f-428c-4276-ad91-ddb10b8900d5.jpeg)
![Screenshot PP](https://user-images.githubusercontent.com/74778466/169098605-fb39eae5-a24c-4d64-a18e-0b4f752d81d8.jpeg)

There is a music player to enable sounds of the redeclipse 1.0 game, it is written the name, and the app it is in, is to make pixel art, yet nothing is expected.

**Comment by qreeves** [comment]
Topic does not describe a bug or issue with the project, moving to `base` and converting to discussion.

**Comment by qreeves** [comment, marked as answer]
The Red Eclipse soundtrack is [Creative Commons Attribution-ShareAlike](https://github.com/redeclipse/sounds/blob/master/music/readme.txt). Most content in the game [is provided under this](https://github.com/redeclipse/base/blob/master/doc/license.txt), or a similar license. The only thing you can't use without permission is our name/logos/etc, as per our [trademark policy](https://github.com/redeclipse/base/blob/master/doc/trademark.txt).

**Reply by vipertechofficial** [reply]
Awesome! https://github.com/pixa-pics/pixa-pics.github.io/tree/main/src/sounds/music/redeclipse  changed with success.
